wolfcrypt/test/test.c

```diff
@@ -6259,6 +6259,7 @@ static int ecc_test_vector_item(const eccVector* vector, WC_RNG* rng)
     memset(sig, 0, sizeof(sig));
     x = sizeof(sig);
 
+    wc_ecc_free(&userA);  /* free key before importing a new one over the top */
```

Comment:
Looking at this again and wc_ecc_make_key and wc_ecc_check_key do not need to be here. Which also means this new wc_ecc_free doesn't need to be here. Lines 6251 through 6262 should just be removed.